### ISSUE TITLE
How often will the app fetch for test results in version 2.1?

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-documentation#585]
## Your Question

* Documentation File: n/a
* Line / Paragraph: n/a
* Repository: https://github.com/corona-warn-app/cwa-quick-test-backend & https://github.com/corona-warn-app/cwa-quick-test-frontend
* Question: Version 2.1 will include a feature which makes it possible to register [quick tests](https://github.com/corona-warn-app/cwa-app-ios/pull/2380) in the app. How often will the app fetch for test results in version 2.1?

Background to this question: Quick Test, as their name says, are processed very quick. To deliver any advantage reg. the delivery of the test result via f.e. e-mail, the Corona-Warn-App should be min. the same speed as the e-mail.
IIRC the app looks for test result all 2 hours currently, this would not be enough for quick tests.

**Comment by thomasaugsten** [comment]
We will try to schedule in the 1h after test registration to check  more often but this is still in development

**Comment by Ein-Tim** [comment]
@thomasaugsten 

Okay thanks, I'll raise this question again when the release of version 2.1 is nearer. 👍

**Comment by Ein-Tim** [comment]
Can you give an answer now @thomasaugsten?

**Comment by MikeMcC399** [comment]
The FAQ https://www.coronawarn.app/en/faq/#rat_how_get_result does not specify any exact polling interval:

"To get the result in the app as soon as possible, you can refresh the test in the app until the result is available. Although the Corona-Warn-App also retrieves test results automatically in the background, it may take a little longer depending on your operating system."

**Comment by MikeMcC399** [comment]
@Ein-Tim

On Android it looks like it does polling at 15 minute intervals for the first 90 minutes, thereafter polling at 90 minute intervals.

I'm going by reading the two files:
- https://github.com/corona-warn-app/cwa-app-android/blob/release/2.1.x/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/rapidantigen/execution/RAResultScheduler.kt
- https://github.com/corona-warn-app/cwa-app-android/blob/release/2.1.x/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/rapidantigen/execution/RAResultRetrievalWorker.kt

It would be good to have confirmation that I read this correctly. Also to know what the scheme on iOS is.

**Comment by thomasaugsten** [comment]
This is correct Android will reduce the fetching but no different polling on iOS side

**Comment by Ein-Tim** [comment]
Thank you for the clarification @thomasaugsten & @MikeMcC399!

Closing as answered.
